docs/visual-basic/misc/bc30282.md

```diff
@@ -45,4 +45,4 @@ A call to `New()` occurs after the first statement of a constructor. If one cons
 -   Remove the call to `New()`, or move it to the beginning of the constructor.  
   
 ## See Also  
- [NOT IN BUILD: Using Constructors and Destructors](http://msdn.microsoft.com/en-us/548eebe1-86c4-4377-b2f5-447cb8be3d90)
\ No newline at end of file
+ [Constructors](~/docs/visual-basic/programming-guide/concepts/object-oriented-programming.md#constructors)
```

Comment:
No, this was an issue with the anchors on the topic. My last commit should fix this.